neurips2022/track2/submission/policy.py

```diff
@@ -5,8 +5,9 @@
 import math
 import pickle
 import torch
+import sys
 from torchvision import transforms
-from model_IL import MainNet
+from submission.model_IL import MainNet
```

Comment:
Does this code work (i) locally, (ii) inside Docker container, and also (iii) in Codalab?